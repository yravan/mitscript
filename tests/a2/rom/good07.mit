x = 0;
a = fun(str) {
  global x;
  print(str + ": " + x);
  x = x + 1;
};

r = {s: a("s"); b: a("b");};
