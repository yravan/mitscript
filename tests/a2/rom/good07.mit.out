s: 0
b: 1
